internal/daemon/daemon.go

```diff
@@ -354,6 +357,13 @@ func logit(handler http.Handler) http.Handler {
 // Init sets up the Daemon's internal workings.
 // Don't call more than once.
 func (d *Daemon) Init() error {
+	if _, err := d.overlord.ServiceManager().Plan(); err != nil {
+		return err
+	}
```

Comment:
Per above, these three lines could be moved to a `Validate` method, and the `dry` field removed.